docs/examples/backup/minimal_appbinding.yaml

```diff
@@ -10,3 +10,4 @@ spec:
       port: 5432
   secret:
     name: my-database-credentials-secret
+  type: postgres
```

Comment:
Is this for manually writing?

Reply:
user can provide anything. it is just a sample.

Reply:
added comment to clarify that this field is optional and user can keep it empty.